idaes/core/util/tags.py

```diff
@@ -223,12 +242,6 @@ def get_unit_str(self, index=None):
     def is_var(self):
         """Whether the tagged expression is a Pyomo Var. Tagged variables
         can be fixed or set, while expressions cannot.
-
```

Comment:
Why did you delete the documentation here? It is rare for more documentation to be bad thing.

Reply:
It's a property.  It used to be a function, but I forgot to fix the docstring.

Reply:
Ah - Github conveniently cut off the display right at the `def` line so I didn't see that.